columnflow/plotting/plot_util.py

```diff
@@ -17,6 +17,51 @@
 od = maybe_import("order")
 
 
+def apply_variable_settings(hists: dict, variable_settings: dict | None = None) -> dict:
```

Comment:
Could you perhaps add a docstring? Just to make it easier to dive in.

Reply:
sure!